test_pytest/test_multipatch.py

```diff
@@ -0,0 +1,203 @@
+import numpy as np
+
+import splinepy
+
+
+class SplineSetup:
```

Comment:
we only use `list_of_splines` and `self_referencing_spline` for the test, so can we just create 2 fixtures?

Reply:
done